app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -19,6 +18,8 @@ import com.azavea.rf.utils.queryparams.QueryParametersCommon
 import com.azavea.rf.scene.SceneQueryParameterDirective
 
 
+case class BulkSceneOperation (action: String, sceneIds: Seq[UUID])
```

Comment:
This might be going away if we switch to HTTP verbs only as per @notthatbreezy's comment below, but as a matter of general principle: since this case class was only required for the route interaction, and not directly the datamodel, I felt like it belonged more here than in the datamodel.